server/routes/addressLookup.ts

```diff
@@ -33,6 +33,5 @@ export default {
   },
   learningPlan: {
     profile: (id: string, tab = 'work-and-interests') => `${config.learningPlanUrl}/plan/${id}/view/${tab}`,
-    addGoals: (id: string) => `${config.learningPlanUrl}/plan/${id}/goals/create`,
```

Comment:
I've removed the `addGoals` property from the `addressLookup`
This property is the URL for where we redirect to at the end of creating an induction. I was never a fan of this file and address lookup approach - the property name is very much tied to it's destination rather than its use case; and I see no purpose or reason to have this as a lookup (over and above simply having the URL in the controller, which is what I've done)